lib/config/config.go

```diff
@@ -215,8 +215,9 @@ type FolderDeviceConfiguration struct {
 
 type OptionsConfiguration struct {
 	ListenAddress           []string `xml:"listenAddress" json:"listenAddress" default:"tcp://0.0.0.0:22000"`
-	GlobalAnnServers        []string `xml:"globalAnnounceServer" json:"globalAnnounceServers" json:"globalAnnounceServer" default:"udp4://announce.syncthing.net:22027, udp6://announce-v6.syncthing.net:22027"`
+	GlobalAnnServers        []string `xml:"globalAnnounceServer" json:"globalAnnounceServers" json:"globalAnnounceServer" default:"https://announce.syncthing.net, https://announce-v6.syncthing.net"`
```

Comment:
Do you actually have a certificate to back it up?
Should we do something like `https://add.somehost.com/?id=<cert hash>` for those that don't?